modular_skyrat/modules/company_imports/code/armament_datums/deforest_medical.dm

```diff
@@ -180,6 +180,27 @@
 	item_type = /obj/item/hypospray/mkii
 	cost = PAYCHECK_LOWER * 2
 
+// Cybernetics and other things that go inside you
+
+/datum/armament_entry/company_import/deforest/implant
+	subcategory = "Cybernetics"
+
+/datum/armament_entry/company_import/deforest/implant/surgery
+	item_type = /obj/item/organ/internal/cyberimp/arm/surgery
+	cost = PAYCHECK_COMMAND * 10
```

Comment:
Honestly, they don't need to cost a 1000, there's a lot worse you can do eitherway